mavros/753226d/753226d.bug

```diff
@@ -74,4 +74,4 @@ fault-codes:
   - CONCURRENCY:TIMING
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```

Comment:
Seems like another case of BEHAVING-INCORRECTLY: "Therefore, any subscribers
  which rely on this timing information will behave incorrectly."